.github/workflows/common-slack-notify-rerequested.yml

```
@@ -0,0 +1,73 @@
+          PR_REVIEWER: ${{ github.event.requested_reviewer.login }}
+          REPOSITORY: ${{ github.repository }}
+        run: |
+          declare -A GITHUB_TO_SLACK
+          GITHUB_TO_SLACK["soeun2537"]="U09A0LM0CRW"
+          GITHUB_TO_SLACK["taek2222"]="U099ARRH3D3"
+          GITHUB_TO_SLACK["changuii"]="U099BR9RNE6"
+          GITHUB_TO_SLACK["eoehd1ek"]="U0995NANDML"
+          GITHUB_TO_SLACK["oungsi2000"]="U098U2R57NK"
+          GITHUB_TO_SLACK["parkjiminnnn"]="U098U8SLXHD"
+          GITHUB_TO_SLACK["etama123"]="U0995MPSZ62"
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)